va/va_enc_hevc.h

```diff
@@ -316,6 +316,9 @@ typedef struct _VAEncSequenceParameterBufferHEVC {
     uint8_t     max_bytes_per_pic_denom;
     /** \brief Same as the HEVC bitstream syntax element. */
     uint8_t     max_bits_per_min_cu_denom;
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                __reserved[VA_PADDING_LOW];
```

Comment:
It will be better to reserve more bytes for HEVC.
For the SPS parameter: it can be PADDING_MEDIA.